### PR TITLE
Cer lag subifc docs

### PR DESCRIPTION
**Pull request by ataberkurn**

### CODE REVIEW

File: frinx-uniconfig/translation-units-docs/configuration-datasets/interfaces/lag_interface.md [hunk 1]
```diff
@@ -596,3 +599,21 @@ there is not possible to set vlan-untagged-data and vtag-stack both.
...
+
+<pre>
+ interface {{subifc_name}}
+  isis wide-metric {{value}} {{level_type}}
+  ip ospf cost {{cost}}
+</pre>
+
+{{value}} is conversion of {{metric_value}}  
+{{level_type}} is conversion of {{level_type}}   
+{{cost}} is conversion of {{ospf_cost}}
```
Comment (mpastorek) [inline frinx-uniconfig/translation-units-docs/configuration-datasets/interfaces/lag_interface.md:615]: remove it

File: frinx-uniconfig/translation-units-docs/configuration-datasets/interfaces/lag_interface.md [hunk 2]
```diff
@@ -108,8 +108,11 @@ frinx-openconfig-interfaces:interfaces/interface={{lag_ifc_name}}
                             "description": "{{lag_sub_description}}",
                             "frinx-juniper-if-extension:rpm-type": "{{rpm_type}}",
                             "frinx-if-aggregate-extension:subinterface-name": "{{subifc_name}}",
-                            "enabled": {{lag_sub_enabled}}
-                        },
+                            "enabled": {{lag_sub_enabled}},
+                            "frinx-cer-if-aggregate-extension:metric": "{{metric_value}}",
+                            "frinx-cer-if-aggregate-extension:level-type": "{{level_type}}",
+                            "frinx-cer-if-aggregate-extension:cost": "{{ospf_cost}}"
+                        ,
```
Comment (mpastorek) [inline frinx-uniconfig/translation-units-docs/configuration-datasets/interfaces/lag_interface.md:115]: } is missing

File: frinx-uniconfig/translation-units-docs/configuration-datasets/interfaces/lag_interface.md (hunk 1 above)
Comment (ataberkurn) [reply frinx-uniconfig/translation-units-docs/configuration-datasets/interfaces/lag_interface.md]: done

File: frinx-uniconfig/translation-units-docs/configuration-datasets/interfaces/lag_interface.md (hunk 2 above)
Comment (ataberkurn) [reply frinx-uniconfig/translation-units-docs/configuration-datasets/interfaces/lag_interface.md]: done

**Review by mpastorek — APPROVED** [review]

**Pull request merged**